tests/e2e-test.spec.js

```diff
@@ -31,108 +38,264 @@ const waitForDbt = async (pgClient, retry = 30) => {
+      expect(modelContactResult.rows.length).to.equal(0);
 
-    await deleteDoc(person);
-    await delay(6); // wait for CHT-Sync
-    await delay(12); // wait for DBT
+      const postDelete = await client.query(`SELECT * FROM ${pgSchema}.persons where uuid = $1`, [person._id]);
+      expect(postDelete.rows.length).to.equal(0);
+    });
 
-    const modelContactResult = await client.query(`SELECT * FROM ${pgSchema}.contacts where uuid = $1`, [person._id]);
-    expect(modelContactResult.rows.length).to.equal(0);
+    it('should process report deletes', async () => {
+      const report = reports()[0];
+      await deleteDoc(report);
+      await delay(6); // wait for CHT-Sync
+      await delay(6); // wait for DBT
+      const pgTableReport = await client.query(`SELECT * from ${PGTABLE} where _id = $1`, [report._id]);
+      expect(pgTableReport.rows[0]._deleted).to.equal(true);
+      const modelReportResult = await client.query(`SELECT * FROM ${pgSchema}.reports where uuid = $1`, [report._id]);
+      expect(modelReportResult.rows.length).to.equal(0);
+    });
 
-    const postDelete = await client.query(`SELECT * FROM ${pgSchema}.persons where uuid = $1`, [person._id]);
-    expect(postDelete.rows.length).to.equal(0);
+    it('should process incremental inserts', async () => {
+      const newDoc = {
+        _id: uuidv4(),
+        type: 'contact',
+        name: 'New Contact',
+        phone: '1234567890',
+        parent: { _id: uuidv4() },
+      };
+
+      await insertDoc([newDoc]);
+      await delay(6); // wait for CHT-Sync
+      await delay(12); // wait for DBT
+
+      const pgTableNewDoc = await client.query(`SELECT * from ${PGTABLE} where _id = $1`, [newDoc._id]);
+      expect(pgTableNewDoc.rows.length).to.equal(1);
+      const modelNewDocResult = await client.query(`SELECT * FROM ${pgSchema}.contacts where uuid = $1`, [newDoc._id]);
+      expect(modelNewDocResult.rows.length).to.equal(1);
+      expect(modelNewDocResult.rows[0].name).to.equal(newDoc.name);
+    });
   });
 
-  it('should process report deletes', async () => {
-    const report = reports()[0];
-    await deleteDoc(report);
-    await delay(6); // wait for CHT-Sync
-    const pgTableReport = await client.query(`SELECT * from ${PGTABLE} where _id = $1`, [report._id]);
-    expect(pgTableReport.rows[0]._deleted).to.equal(true);
-    await delay(6); // wait for DBT
-    const modelReportResult = await client.query(`SELECT * FROM ${pgSchema}.reports where uuid = $1`, [report._id]);
-    expect(modelReportResult.rows.length).to.equal(0);
+  describe('Conflict resolution', () => {
+    it('should handle update conflicts', async () => {
+      const contact = contacts()[2];
+
+      const editedContactA = { ...contact, edited: 'A' };
+      const editedContactB = { ...contact, edited: 'B' };
+
+      await editDoc(editedContactA);
+      await delay(1);
+      await editDoc(editedContactB);
```

Comment:
This will never create a conflict, because of how `editDoc` is written, so the name of the test is misleading. 
To create a conflict, you need to push the doc with an old rev and `new_edits=true`. 

Details here: 
https://docs.couchdb.org/en/stable/api/document/common.html#put--db-docid